br/pkg/backup/client.go

```diff
@@ -282,15 +282,34 @@ func BuildBackupRangeAndSchema(
 	storage kv.Storage,
 	tableFilter filter.Filter,
 	backupTS uint64,
-) ([]rtree.Range, *Schemas, error) {
+	isFullBackup bool,
+) ([]rtree.Range, *Schemas, []*backuppb.Policy, error) {
 	snapshot := storage.GetSnapshot(kv.NewVersion(backupTS))
 	m := meta.NewSnapshotMeta(snapshot)
 
+	var policies []*backuppb.Policy
+	if isFullBackup {
```

Comment:
Need more code to ignore placement options in table/schema when not full backup?

> Otherwise, While backing up the specified database/table (using br backup db/table or [table filter](https://docs.pingcap.com/tidb/stable/use-br-command-line-tool#back-up-with-table-filter)), placement policies are not backed up. Placement options in table/database will not be backed up too.

Reply:
fixed in https://github.com/pingcap/tidb/pull/33007/files#diff-e611f02c279107dbfa433418c809e99569c0b24c2ea92cc7aef9f7f41fbcc3b9R361